README.rst

```diff
@@ -60,6 +60,9 @@ Usage
     servers = await nova.servers.list(name='testvm')
     vm = await nova.servers.get(server_id)
 
+    action_spec = {'os-stop': None}
+    await nova.servers.run_action(server_id, **action_spec)
```

Comment:
Maybe I'm missing the point, but I don't think unpacking `action_spec` to kwargs is desirable behavior here. Maybe you meant `action= action_spec` as stated later?

Reply:
The problem is that `await nova.servers.run_action(server_id, os-stop=None)` is not valid Python. Unpacking does the trick. Somehow ugly, but the only way to send `{'os-stop': null}` to OpenStack.

Reply:
Gotcha, I though it would be similar to volumes api, that you pass whole chunk to one `volume` kwarg, didn't read the docs tho 😉 Looks good to me then